src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Address.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Address]
+(
+	AddressId INT IDENTITY(1,1) NOT NULL PRIMARY KEY, 
+    Line1 NVARCHAR(256) NULL, 
+    Line2 NVARCHAR(256) NULL, 
+    Line3 NVARCHAR(256) NULL, 
+    Line4 NVARCHAR(256) NULL, 
+    Line5 NVARCHAR(256) NULL, 
+    Town NVARCHAR(256) NULL, 
+    County NVARCHAR(256) NULL, 
+    Postcode NVARCHAR(10) NULL, 
+    Country NVARCHAR(256) NULL

```

Comment:
Personally, I'd prefer lower case data types but I'm not sure we have any agreed standards on SQL formatting (yet) so your choice for now! 😃

Reply:
I'm sure we can discuss it on Friday 😄